### ISSUE TITLE
Make columnWidthValues part of column array property?

### ISSUE THREAD

**Issue by kasbah** [denisraslov/react-spreadsheet-grid#31]
Think it would be a bit neater to be able to do something like this:

```jsx

class MyAwesomeGrid extends React.Component {
  render() {
    return (
      <Grid 
        columns={[
          {
            title: () => 'Name', 
            width: 100,
            value: (row, { focus }) => {
                return (
                    <Input  
                      value={row.name}
                      focus={focus}
                    />
                );
            }
           ...
```

**Comment by denisraslov** [comment]
Hi @kasbah 
Thank you, it seems like a good idea!
Don't you want to make a PR with the implementation?

**Comment by denisraslov** [comment]
Hi @kasbah
This was implemented in the version 2.0 of the library. You can use it now.
Please, see the docs for more of the information: https://github.com/denisraslov/react-spreadsheet-grid#columns
